### ISSUE TITLE
Context URL Redirect to Producing SSL Error at digitalbazaar.github.io

### ISSUE THREAD

**Issue by kayaelle** [digitalbazaar/ed25519-signature-2020-context#6]
https://w3id.org/security/suites/ed25519-2020/v1 redirects to https://digitalbazaar.github.io/ed25519-signature-2020-context/contexts/ed25519-signature-2020-v1.jsonld

and is resulting in an SSL error causing an invalid JSON-LD object result when attempting to verify a credential.

**Comment by msporny** [comment]
Yeah, Github has screwed up some part of their SSL certificate environment. Note that `digitalazaar.github.io` is served from Github, not DB's servers. We started seeing this issue in Europe, and looks like it's now creeping into the US. We don't have a solution at present to it, we're trying to track down what's going on.

To be clear, whatever software is returning an error should have this context cached. Doing anything other than that will result in spurious errors like this when URLs don't resolve as you expect them to.
